### DISCUSSION TITLE
Setting / Reading Message-ID

### DISCUSSION THREAD

**Discussion by the4thv** [O365/python-o365#738] [Q&A] [answered]
Firstly, thank you for a great library! 

I'm trying to either read the Message-ID from the message object (with the object instance still in memory after send()) or set it / set the conversationId such that I can pick up replies to a specific message programmatically.

I'm using Django with this library and I would usually use ```make_msgid``` function in django.core.mail and then assign that value to the message-id value in the header. 

Could you help please? I've looked into message.py and there does seem to be the possibility to update either message-id or conversationId:

```
msg = mailbox.new_message()
msg.conversation_id = 'foo'
msg.to_api_data()
```
The dict is not being updated.

**Comment by alejcas** [comment, marked as answer]
Currently conversation_id is read only and so is object_id
